GoogleDataTransport/GDTCORTests/Unit/GDTCORStorageTest.m

```diff
@@ -16,13 +16,11 @@
 
 #import "GDTCORTests/Unit/GDTCORTestCase.h"
 
-#import <GoogleDataTransport/GDTCOREvent.h>
-#import <GoogleDataTransport/GDTCORStoredEvent.h>
-
 #import "GDTCORLibrary/Private/GDTCOREvent_Private.h"
 #import "GDTCORLibrary/Private/GDTCORRegistrar_Private.h"
 #import "GDTCORLibrary/Private/GDTCORStorage.h"
 #import "GDTCORLibrary/Private/GDTCORStorage_Private.h"
+#import "GDTCORLibrary/Public/GDTCOREvent.h"
```

Comment:
Separate the blocks of imports by path. Public headers should be in their own block of imports